manifests/profile/neutron/server.pp

```diff
@@ -12,11 +12,40 @@
   $tunnel_id_ranges              = $::openstack::config::neutron_tunnel_id_ranges # ['1:1000']
   $controller_management_address = $::openstack::config::controller_address_management
 
-  class  { '::neutron::plugins::ml2':
-    type_drivers         => $type_drivers,
-    tenant_network_types => $tenant_network_type,
-    mechanism_drivers    => $mechanism_drivers,
-    tunnel_id_ranges     => $tunnel_id_ranges
+  if ($::openstack::config::neutron_core_plugin == 'ml2') {
```

Comment:
@cmurphy Are you okay with plugins entry point names used here or you still want me to use the whole class name?